src/__snapshots__/signpdf.test.js.snap

```diff
@@ -10,4 +10,4 @@ exports[`Test signing expects PDF to be Buffer 1`] = `"PDF expected as Buffer."`
 
 exports[`Test signing expects PDF to contain a ByteRange placeholder 1`] = `"Could not find ByteRange placeholder: /ByteRange [0 /********** /********** /**********]"`;
 
-exports[`Test signing expects a reasonably sized placeholder 1`] = `"Signature exceeds placeholder length: 3224 > 4"`;
+exports[`Test signing expects a reasonably sized placeholder 1`] = `"Signature exceeds placeholder length: 3010 > 4"`;
```

Comment:
This here is what I meant needs checking. The signature is now 214 characters shorter. That is, 107 encoded bytes less. What is missing from the signature? Will it be a regression for someone who's already using the library?
ASN reading will be needed for this to be accepted.